### ISSUE TITLE
`unused-void-attr` Linter Incorrectly Flags Void Attributes in Atoms

### ISSUE THREAD

**Issue by h1alexbel** [objectionary/lints#482]
Here is the code:

```eo
# Error.
[message] > error ?
```

Here, `error` object is an atom, but I'm getting this error:

```text
The void attribute "message" is unused anywhere in the object "error"
```

The lint `unused-void-attr` should not complain about void attributes in atom.

Let's fix it.

**Comment by h1alexbel** [comment]
@yegor256 new bug

**Comment by yegor256** [comment]
@h1alexbel definitely a bug, thanks for reporting

**Comment by h1alexbel** [comment]
this one is solved
